deploy/helm/charts/templates/priority-class.yaml

```diff
@@ -1,15 +1,15 @@
 apiVersion: scheduling.k8s.io/v1
 kind: PriorityClass
 metadata:
-  name: openebs-csi-controller-critical
+  name: openebs-jiva-csi-controller-critical
```

Comment:
The priority class name can come from values.yaml or else defaulted. 
It can be done by writing few lines in _helpers.tpl

Reply:
Also -- Is the priority class created always when the jiva operator is deployed? Can't it be tunable? 
I mean, create priority class only if I want.

Reply:
I know this is coming from the existing code. So we could do this in a different PR.

Reply:
>The priority class name can come from values.yaml or else defaulted.
It can be done by writing few lines in _helpers.tpl

Yes we can write it in __helpers.tpl_. As we are keeping hard coded value I was not thinking of __helpers.tpl_.

Reply:
>Also -- Is the priority class created always when the jiva operator is deployed? Can't it be tunable?
I mean, create priority class only if I want.

I had a offline discussion with @prateekpandey14 on this.
From that meeting - _As storage is a critical component we don't want to make it configurable_.

We can make it configurable :+1:.  Can you please provide some requirements for this.

Reply:
In this case, we are defining the name in one place and using it in another. Does the helm linter catch an error if another contributor comes and just changes the name in one place?

Are there other instances in helm chart where names are hardcoded to prefix with openebs or do we typically use chart full name?

Reply:
>In this case, we are defining the name in one place and using it in another. Does the helm linter catch an error if another contributor comes and just changes the name in one place?

I think no, to avoid this we can use _helpers.tpl

Reply:
>In this case, we are defining the name in one place and using it in another. Does the helm linter catch an error if another contributor comes and just changes the name in one place?
Shovan is correct. Also, we may not be able to use _helpers.tpl at every place. 

>As storage is a critical component we don't want to make it configurable.
I need more understanding here.

Reply:
My understanding is -- by default, we create the priority class. Let the user turn it off if they want to.